docs/reST/ref/surface.rst

```diff
@@ -902,4 +902,26 @@
 
       .. versionadded:: 1.9.2
 
+   .. method:: premul_alpha
```

Comment:
Maybe it should be more explicit about what you are and aren't supposed to do with a premultiplied surface.

Thoughts:
- Pixels are actually mutated, using get_at() won't compensate, you shouldn't draw onto these surfaces?
- Can you blit a premultiplied surface onto another premultiplied surface? I don't think that would work.
- When blitting this, you need to use the BLEND_PREMULTIPLIED flag to get correct results.
- This is for high speed of repeated blitting of static (unchanging) surfaces, right?